R/utilities-path.R

```diff
@@ -23,7 +23,8 @@ toPathArray <- function(path) {
 #'
 #' path <- toPathString(c("Organism", "Organ", "Liver"))
 #' @export
-toPathString <- function(pathArray) {
-  validateIsString(pathArray)
-  paste(pathArray, collapse = ospsuiteEnv$pathSeparator)
+toPathString <- function(...) {

```

Comment:
Little trick so that you do not have to do toPathString(c("A", "B"). Instead it can be toPathString("A", "B")

Reply:
old method still compatible

Reply:
nice